shapely/strtree.py

```diff
@@ -122,28 +129,59 @@ def __reduce__(self):
         else:
             return (STRtree, (self.geometries,))
 
-    def query_items(self, geom: BaseGeometry) -> Sequence[Any]:
-        """Query for nodes which intersect the geom's envelope to get
-        stored items.
+    def query_items(
+        self, geom: BaseGeometry, predicate=None, distance=None
+    ) -> Sequence[Any]:
+        """
+        Return the index of all geometries in the tree with extents that
+        intersect the envelope of the input geometry.
 
         Items are integers serving as identifiers for an application.
 
+        If predicate is provided, a prepared version of the input geometry
+        is tested using the predicate function against each item whose
+        extent intersects the envelope of the input geometry:
+        predicate(geometry, tree_geometry).
```

Comment:
I think that to help users who don't know the implementation we could be more clear about which thing is on the left hand side of the predicate and which is on the right side. The input geometry is on the left and the item to be tested is on the right, yes?

Reply:
The line you are commenting on (`predicate(geometry, tree_geometry)` basically shows that "geometry" is left and "tree geometry" is right? (but maybe can make it `predicate(input_geometry, tree_geometry` (i.e. added `input_`) to make it clearer) 
Or that's not super clear this way?

Reply:
Adding `input_` prefix would help here, though it immediately follows (is implicit) from the prior statement.